cms/src/components/content/ContentTable.vue

```diff
@@ -86,6 +87,15 @@ const contentDocs = contentOverviewQueryAsRef(props.queryOptions);
                         />
                     </tbody>
                 </table>
+                <div
+                    class="flex h-32 w-full items-center justify-center gap-2"
+                    v-if="contentDocs.length < 1"
+                >
+                    <ExclamationTriangleIcon class="h-6 w-6 text-zinc-600" />
+                    <p class="text-sm font-extralight text-zinc-600">
+                        No content found with the matched filter.
+                    </p>
```

Comment:
Nice addition! Just some suggested style changes:

```suggestion
                    <ExclamationTriangleIcon class="h-6 w-6 text-zinc-500" />
                    <p class="text-sm text-zinc-500">
                        No content found with the matched filter.
                    </p>
```